build.sbt

```diff
@@ -1,109 +0,0 @@
-organization in ThisBuild := "sample.chirper"
```

Comment:
As @ignasi35 said, disable the PlayLayoutPlugin, and you should be 90% of the way there to being able to build with either sbt or Maven.